src/virtual_dom/node/el.rs

```diff
@@ -9,6 +9,26 @@ use crate::browser::{
 };
 use std::borrow::Cow;
 
+// ------ ElKey ------
+
+#[allow(clippy::module_name_repetitions)]
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub struct ElKey(String);
+
+#[allow(clippy::module_name_repetitions)]
+/// Attach given `key` to the `El`.
+///
+/// ## Warning
+///
+/// WIP
+///  - it doesn't patch elements with the different keys, BUT it replace the old element with the new one.
+///  - `key` restriction (have to implements `ToString`) will be probably relaxed in the future.
```

Comment:
Are these comments still valid? If so, are there interesting for users (i.e. should  it be changed to regular comment)?